configs/default/DIAT-FURYF4.config

```diff
@@ -78,6 +78,7 @@ dma pin A00 0
 # pin A00: DMA1 Stream 2 Channel 6
 
 # master
+set baro_hardware = AUTO
```

Comment:
Sorry, looks like I was not explicit enough - `baro_hardware = AUTO` is a default as well and should not be explicitly used in a Unified Target configuration.

Hint: You can check this yourself, the default is listed for parameters on `get` if they are not already set to the default:

```
# get mag_hardware
mag_hardware = NONE
Allowed values: AUTO, NONE, HMC5883, AK8975, AK8963, QMC5883, LIS3MDL, MAG_MPU925X_AK8963
Default value: AUTO
```

Reply:
Sorry about that.  Misunderstood what you were getting at.   Got it now.   I’ll remove completely and push the change once I get back from vacation.  Thanks again.

Reply:
@mikeller Complete.  All removed.  Thanks again.